### DISCUSSION TITLE
¿Qué esta pasando?

### DISCUSSION THREAD

**Discussion by Matiasox** [Launch-X-Latam/MisionFrontEnd#529] [Team Posts]
Por temas de tiempo no he podido ingresar a la primera clase de introducción a JavaScript y veo que están haciendo grupos para una hackaton ¿Quién me pone al día por favor?

**Comment by giankpetrov** [comment]
Yo tambien quiero informarme, nose porque la gente esta haciendo equipos de una vez

**Comment by Amhedriel** [comment]
Por un consejo del buen Sherpa de BackEnd, asi que les recomiendo ver el video de presentación de BackEnd del Lunes para estar más al tanto de todo, saludos colegas.

**Comment by giankpetrov** [comment]
@Amhedriel donde esta el video

**Comment by Amhedriel** [comment]
En Teams

**Comment by Amhedriel** [comment]
No sé si te servirá este link, pero probemos [TEAMS BackEnd](https://teams.microsoft.com/_#/school/tab::be634985-ba21-4017-8458-d6979476ca11/General?threadId=19:59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1@thread.tacv2&ctx=channel)
